subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/TaskErrorOutputCaptureExecutionListener.kt

```diff
@@ -44,6 +44,15 @@ internal class TaskErrorOutputCaptureExecutionListener(
     override fun afterExecute(task: Task, state: TaskState) {
         if (state.failure == null) {
             logs.remove(Path.path(task.path))
+        } else {
+            when (task) {
+                is BuildVerdictTask -> {
+                    task.ciLogger.debug("Add verdict listener to the task ${task.path}")
```

Comment:
```suggestion
                    task.ciLogger.debug("Get a verdict from task ${task.path}")
```
?
I didn't get where is a listener here